topologies/binding/options.go

```diff
@@ -140,6 +141,22 @@ func (d *dialer) newHTTPClient() *http.Client {
 	return &http.Client{Transport: tr}
 }
 
+// newIxWebClient makes an IxWeb session using the binding options.
+func (d *dialer) newIxWebClient(ctx context.Context) (*ixweb.IxWeb, error) {
+	hc := d.newHTTPClient()
+	password := d.GetPassword()
+	username := d.GetUsername()
+	if len(password) == 0 || len(username) == 0 {
```

Comment:
Let's make some changes here if you don't mind.

```
if username == "" && password == "" {
  ...
}
```

Reply:
sure. I will make those changes now

Reply:
changes are done